Partition (WBP)/Functions/V57033F.VT157033.fun.yml

```diff
@@ -183,555 +272,3640 @@
     Add_Entry To_List(#wLTestResultExactSigned_Decimals)
     
     * *** Generic Signed Results
-    #LF_Test_Result_L157033FI1 #LF_Test_Result_L157033FN := 320
-    #LF_Test_Result_L157033FI2 #LF_Test_Result_L157033FI4 #LF_Test_Result_L157033FI8 #LF_Test_Result_L157033FP := 0
-    #LF_Test_Result_L157033FPD #LF_Test_Result_L157033FND := 320
+    #LF_Test_Result_L157033FP := 0
```

Comment:
You need to implement testing for #LF_Test_Result_L157033FP value too before you complete this particular search.

Reply:
Ok..Will be doing it.

Reply:
Thank you